FrontEnd/src/components/ProfilePage/FormComponents/FormFields/CheckBoxField.js

```diff
@@ -0,0 +1,45 @@
+import css from './CheckBoxField.module.css';
```

Comment:
NP that we created custom fields. But designers utilized [ant design](https://ant.design/components/overview) that would simplify it

Reply:
Yes, I heard this from mentors, but I decided to create custom fields. I am beginner in React. But I will keep this in mind in the future. Can i leave it as is? or does it need to be redone?